### PR TITLE
Create model class

### PR DESCRIPTION
**Pull request by polamarysia**

### CODE REVIEW

File: src/skibidi_orm/query_engine/model/base.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+""" A model class for mapping database table """
+
+from pydantic import BaseModel
+from pydantic.main import ModelMetaclass
+from typing import Any, Tuple, Dict, List
```
Comment (msobol12) [inline src/skibidi_orm/query_engine/model/base.py:5]: you can write typehints without importing from typing Tuple, Dict, List. You can write dict[str, str] and it works. You don't have to change it, this is ok too.

File: src/skibidi_orm/query_engine/model/base.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    return True
+
+def _is_field(value: Any):
+    # Only call contribute_to_class() if it's bound.
+    return not inspect.isclass(value) and hasattr(value, "contribute_to_class")
+
+class MetaOptions:
+    """ A subsidiary to storage additional information"""
+    def __init__(self, meta: Dict[str, Any]):
+        self.fields = []
```
Comment (msobol12) [inline src/skibidi_orm/query_engine/model/base.py:22]: I can't find where you use it. Maybe you forgot to remove it, it looks like duplicate of local_field

File: src/skibidi_orm/query_engine/model/base.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+def check_db_name_is_correct(db_table_name: str) -> bool:
+    return True
+
+def _is_field(value: Any):
+    # Only call contribute_to_class() if it's bound.
+    return not inspect.isclass(value) and hasattr(value, "contribute_to_class")
+
+class MetaOptions:
+    """ A subsidiary to storage additional information"""
```
[the remark this answers is not in the document]
Comment (polamarysia) [reply src/skibidi_orm/query_engine/model/base.py:20]: Ok, it is good change

File: src/skibidi_orm/query_engine/model/base.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        self.model = cls
+
+    def add_field(self, field: Any):
+        """ Adds database column """
+        bisect.insort(self.local_fields, field) # type: ignore
+        self.setup_pk(field)
+
+    def setup_pk(self, field: Any):
+        """ Setup the primary key """
+        if self.pk and field.primary_key:
```
[the remark this answers is not in the document]
Comment (polamarysia) [reply src/skibidi_orm/query_engine/model/base.py:39]: Thanks, it is typo

File: src/skibidi_orm/query_engine/model/base.py (hunk 1 above)
Comment (polamarysia) [reply src/skibidi_orm/query_engine/model/base.py:5]: Ok, I will change

File: src/skibidi_orm/query_engine/model/base.py (hunk 2 above)
Comment (polamarysia) [reply src/skibidi_orm/query_engine/model/base.py:22]: Yeah, i will remove it

**Review by msobol12 — APPROVED** [review]

**Pull request merged**